### DISCUSSION TITLE
Add constants for default icons

### DISCUSSION THREAD

**Discussion by skttl** [umbraco/Umbraco-CMS#11137] [Features and ideas]
Following this tweet https://twitter.com/jjowensdev/status/1439863102439411712, could we add those to core? Would make it a lot easier to reference icons in code (when setting up sections, trees etc.)

**Comment by abjerner** [comment]
While not a complete list, there is already this: https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Core/Constants-Icons.cs

**Comment by nul800sebastiaan** [comment]
First reaction: I don't think we should, it will never be a complete list since people can install custom icons. If there's a nice way to build a list of available icons (think: modelsbuilder) then.. maybe.

**Comment by nathanwoulfe** [comment]
Maybe a starting point?
https://github.com/umbraco/Umbraco-CMS/blob/405ed44bb18ab6f5de3b2372b511497470198f89/src/Umbraco.Web/Services/IconService.cs#L29-L32
This gets all known icons (including third-party/user-added), where the key is the icon filename (less the extension). How that would be converted into anything useful, I don't know, but we do have an object containing a reference to every icon. Maybe a source generator?
